src/main/java/org/broadinstitute/hellbender/tools/spark/sv/RunSGAViaProcessBuilderOnSpark.java

```diff
@@ -511,6 +511,22 @@ public String toString(){
             public String toString(){
                 return id;
             }
+
+            @Override
+            public boolean equals(final Object o) {
+                if (this == o) return true;
+                if (o == null || getClass() != o.getClass()) return false;
+
+                final ContigID contigID = (ContigID) o;
+
+                return id != null ? id.equals(contigID.id) : contigID.id == null;
```

Comment:
No big deal, but there's a JDK method for this if you'd prefer it:  `return Objects.equals(id, contigID.id);`